src/agent/src/rpc.rs

```diff
@@ -1682,7 +1682,10 @@ fn is_signal_handled(proc_status_file: &str, signum: u32) -> bool {
                 warn!(sl!(), "parse the SigCgt field failed");
                 return false;
             }
-            let sig_cgt_str = mask_vec[1];
+            let mut sig_cgt_str = mask_vec[1];
```

Comment:
I wonder if we should introduce a function here for parsing the field (maybe starting with [1679](https://github.com/kata-containers/kata-containers/pull/4252/files#diff-f87c1c212bba3abd0b07bafe69d5b01ad00dfe2c01bed585e2a75fcbb6442b89L1679)?), which would allow us to be able to add same unit tests?

Reply:
This should be covered by `test_is_signal_handled` already, let me know if you would prefer to split the function down and change the related tests